webapp/src/components/expanded_view/component.tsx

```diff
@@ -506,22 +508,40 @@ export default class ExpandedView extends React.PureComponent<Props, State> {
         let header = CallRecordingDisclaimerStrings[isHost ? 'host' : 'participant'].header;
         let body = CallRecordingDisclaimerStrings[isHost ? 'host' : 'participant'].body;
         const confirmText = isHost ? 'Dismiss' : 'Understood';
+        let icon = (
+            <RecordCircleOutlineIcon
+                size={18}
+            />);
 
         if (hasRecEnded) {
             header = 'Recording has stopped. Processing...';
             body = 'You can find the recording in this call\'s chat thread once it\'s finished processing.';
         }
 
+        let error = '';
+        if (this.props.callRecording?.err) {
+            header = 'Something went wrong with the recording!';
```

Comment:
```suggestion
            header = 'Something went wrong with the recording';
```

Reply:
Do we know when this error would come up - is it possible to get this in the middle of a call that's being recorded? If so, what happens to the recording up until then? Is it saved or discarded?

Reply:
The way it's implemented so far the error would only show upon starting/stopping the recording. But that's a limitation since technically the recording process could fail at any time (although it would be a rare event). In such cases the file would likely be discarded until we put in place a more advanced system (past beta phase).